docs/keosd/how-to-guides/how-to-attach-a-YubiHSN.md

```diff
@@ -0,0 +1,69 @@
+## Goal
+
+Attach a YubiHSM as a hard wallet
+
+## Before you begin
+
+* Install the currently supported version of kleos
```

Comment:
`kleos` is `cleos` and `keosd`?

Reply:
also elsewhere it has backticks which I think it should have here unless we want to refer to a product suite